generate-jenkins-changelog.rb

```diff
@@ -114,7 +115,14 @@
 root['date'] = Date.parse(`git log --pretty='%ad' --date=short #{new_version}^..#{new_version}`.strip)
 root['changes'] = issues
 
-puts [root].to_yaml
+changelog_yaml = [root].to_yaml
 hidden.sort { |a, b| a['pull'] <=> b['pull'] }.each do | entry |
-	puts "  # pull: #{entry['pull']} (#{entry['message']})"
+	changelog_yaml += "\n  # pull: #{entry['pull']} (#{entry['message']})"
 end
+puts changelog_yaml
+
+changelog_path = ENV["CHANGELOG_YAML_PATH"]
```

Comment:
For consistency with the previous behavior would be preferable as an optional additional argument.

Alternatively, and this seems preferable, shouldn't this be the job of the invoker, i.e. `docker-runner.rb`?

Reply:
Then you would need to parse the CLI output there which is not fun.

Regarding optional arguments, it requires adding a CLI arguments framework to do it well. I could do it, but IMHO it is a separate story since the tool already uses environment variables in the script. Subject for further improvement